whois-commons/src/main/java/net/ripe/db/whois/common/rpsl/DummifierRC.java

```diff
@@ -0,0 +1,75 @@
+package net.ripe.db.whois.common.rpsl;
+
+import com.google.common.collect.Lists;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+@Component
+public class DummifierRC extends DummifierCurrent {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(DummifierRC.class);
```

Comment:
Minor: the logger is not being used to log anything though. Consider using it, or removing

Reply:
I removed it